### PR TITLE
Hot fix a description of over Table API document.

### PR DESCRIPTION
**Pull request by sunjincheng121**
Thanks for contributing to Apache Flink. Before you open your pull request, please take the following check list into consideration.
If your changes take all of the items into account, feel free to open your pull request. For more information and/or questions please refer to the [How To Contribute guide](http://flink.apache.org/how-to-contribute.html).
In addition to going through the list, please provide a meaningful description of your changes.

- [ ] General
  - The pull request references the related JIRA issue ("Hot fix a description of over Table API document.")
  - The pull request addresses only one issue
  - Each commit in the PR has a meaningful commit message (including the JIRA id)

- [x] Documentation
  - Documentation has been added for new functionality
  - Old documentation affected by the pull request has been updated
  - JavaDoc for public methods has been added

- [ ] Tests & Build
  - Functionality added by the pull request is covered by tests
  - `mvn clean verify` has been executed successfully locally or a Travis build has passed

### CODE REVIEW

File: docs/dev/table/tableApi.md [hunk 1]
```diff
@@ -1338,7 +1338,10 @@ The `OverWindow` defines a range of rows over which aggregates are computed. `Ov
 
         <p>At the moment, over windows with rows following the current row are not supported. All over windows must stop at the current row and only two values are supported for <code>following</code>, <code>CURRENT_RANGE</code> for a time interval and <code>CURRENT_ROW</code> for a row-count interval.</p>
 
-        <p>If the `following` clause is omitted, the window will end at the current row.</p>
+       <p><code>CURRENT_ROW</code> meants the window will end at the current row.</p>
```
Comment (zentol) [inline docs/dev/table/tableApi.md]: typo: meants -> means

**Comment by sunjincheng121** [comment]
@zentol  thanks ! :) I have updated the PR.

File: docs/dev/table/tableApi.md [hunk 2]
```diff
@@ -1338,7 +1338,10 @@ The `OverWindow` defines a range of rows over which aggregates are computed. `Ov
 
         <p>At the moment, over windows with rows following the current row are not supported. All over windows must stop at the current row and only two values are supported for <code>following</code>, <code>CURRENT_RANGE</code> for a time interval and <code>CURRENT_ROW</code> for a row-count interval.</p>
 
-        <p>If the `following` clause is omitted, the window will end at the current row.</p>
+       <p><code>CURRENT_ROW</code> means the window will end at the current row.</p>
+       <p><code>CURRENT_RANGE</code> means the window will end at the current row’s last peer in the ORDER BY ordering.</p>
```
Comment (fhueske) [inline docs/dev/table/tableApi.md]: `CURRENT_ROW` sets the upper bound of the window to the current row.
`CURRENT_RANGE` sets the upper bound of the window to sort key of the the current row, i.e., all rows with the same sort key as the current row are included in the window.

**Comment by fhueske** [comment]
merging

**Pull request closed without merging**